### ISSUE TITLE
Get network output during training.

### ISSUE THREAD

**Issue by ilyes-hm** [tensorflow/tensorflow#36470]
Hello,
Is it possible to run `tensor.eval()` in the loss function to get the network output and use it in an external file to compute the loss.
 
Thank you

**Comment by gadagashwini** [comment]
@ilyes-hm, Can you elaborate the issue with sample code and/or any use case. Thanks!

**Comment by ilyes-hm** [comment]
So I am trying to build a loss function that uses the network output and pass it to a command line function to create two matrices then i compute the loss based on these matrices, my problem i cant generate these matrices when I pass the network outputs using `tensor.eval()` to a string to run the command line.
```python
def command(params):   
     cmd = 'a command line'+str(params.eval())
     mat = os.system(cmd)
     return mat
def custom_loss(y_true, y_pred):
    mat1 = command(y_true)
    mat2 = command(y_pred)
    loss = mean_squared_error(mat1,mat2)
    return loss
```

**Comment by gadagashwini** [comment]
@ilyes-hm, Could you provide full error log and complete code to analyze the issue. Thanks

**Comment by ilyes-hm** [comment]
Let me reformulate my question, Is it possible to get the network outputs as numpy and compute the loss using a numpy function (which include a command line that takes the network output as argument)?

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
 Thanks!
